components/script/dom/eventtarget.rs

```diff
@@ -731,10 +731,10 @@ impl EventTarget {
         listener: Option<Rc<EventListener>>,
         options: EventListenerOptions,
     ) {
-        let ref listener = match listener {
+        let listener = &(match listener {
             Some(l) => l,
             None => return,
-        };
+        });
```

Comment:
The compilation succeeds without the `if` keyword. Otherwise, I get a `this 'if' expression is missing a block after the condition` error. Is a `let...else` statement appropriate in this scenario?